apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -354,25 +355,8 @@ function initializeBlocklyWrapper(blocklyInstance) {
   ) {
     if (!this.unusedSvg_) {
       this.unusedSvg_ = new BlockSvgUnused(this, helpClickFunc);
+      this.unusedSvg_.render(this.svgGroup_, this.RTL);
     }
-    this.unusedSvg_.render(this.svgGroup_, this.RTL);
-  };
-
-  const googleBlocklyRender = blocklyWrapper.BlockSvg.prototype.render;
-  blocklyWrapper.BlockSvg.prototype.render = function (opt_bubble) {
-    googleBlocklyRender.call(this, opt_bubble);
-    this.removeUnusedBlockFrame();
-  };
-
-  // The original Google Blockly dispose() is defined at:
-  // https://github.com/google/blockly/blob/1f862cb878f7eec36b71c638b85d5199bff01fcb/core/block_svg.ts#L863
-  const googleBlocklyDispose = blocklyWrapper.BlockSvg.prototype.dispose;
-  // if param healStack is true, then tries to heal any gap by connecting the next
-  // statement with the previous statement
-  // if param animate is true, shows a disposal animation and sound
-  blocklyWrapper.BlockSvg.prototype.dispose = function (healStack, animate) {
```

Comment:
Can you elaborate on how you know it's safe to remove this? I don't see much discussion of this function in the description (mostly focuses on render)? I think I see where you're coming from looking at the Blockly docs [here](https://developers.google.com/blockly/reference/js/blockly.blocksvg_class.dispose_1_method.md) and looking at the way we call dispose in our code, but just wanted to confirm.

Reply:
Oh, yes sorry for the omission! Currently, we are overriding here explicitly to dispose the unused block frame when a block is disposed. Because the new listener also checks for `event.type === Blockly.Events.BLOCK_DELETE` it's taking on the role of both overrides (dispose and render). By removing both overrides, we're letting Blockly handle all block rendering and disposing as it normally would.